tests/test_gnn_ops.py

```diff
@@ -15,15 +16,34 @@ def __init__(self, in_dim, prop_dim, s, k):
             # Ensure that any additional arguments are TorchScript compatible
             optimization_arguments={}
         )
+        self.scripted_gravnet = torch.jit.script(self.gravnet)
         self.fc = torch.nn.Linear(prop_dim, prop_dim)
 
     def forward(self, x, row_splits):
-        x, *_ = self.gravnet(x, row_splits)
+        x, *_ = self.scripted_gravnet(x, row_splits)
         x = self.fc(x)
         return x
 
 class TestGravNetOp(unittest.TestCase):
 
+    def test_jit_script_compatibility(self):
```

Comment:
This is good. But I think it would be good to have similar unit tests for all of the individual extensions; so is something breaks, we know where